test/README.md

```diff
@@ -242,4 +241,18 @@ A few operations are done on the output of a test before the comparison with `TE
 - DMD's debug message (e.g. `DMD v2.084.0-255-g86b608a15-dirty DEBUG`) gets stripped away
 - paths to `test_results` will be replaced with `{{RESULTS_DIR}}`
 
+`TEST_OUTPUT` accepts format specifiers for matching compilation output that is indeterminate
+between compiles or platforms. Recognized specifiers are:
+
+    $n$     Matches any digit "[0-9]"
+    $h$     Matches any hexdigit "[0-9A-Fa-f]"
+
+Example of using format specifiers:
```

Comment:
Many Markdown (and "standard" Markdown) dialects require a new line here.

Reply:
Are we reinventing regex?
https://www.rexegg.com/regex-quickstart.html
I don't mean we need to support regex here, but when adding patterns to match, if we follow regex conventions, we can add patterns as needed without needing to design anything or write a library to match it.

Reply:
In a way yes, that's what the initial addition of `$n$` was doing.  Treating the TEST_OUTPUT as a regex to match I imagine would be problematic as all special regex characters in test sources suddenly need escaping.